pkg/tool/helm.go

```diff
@@ -21,15 +24,47 @@ func NewHelm(exec exec.ProcessExecutor, extraArgs []string) Helm {
 	}
 }
 
+func (h Helm) RunProcessAndCaptureOutput(executable string, execArgs ...interface{}) (string, error) {
+	return h.RunProcessInDirAndCaptureOutput("", executable, execArgs)
+}
+
+func (h Helm) RunProcessInDirAndCaptureOutput(workingDirectory string, executable string, execArgs ...interface{}) (string, error) {
+	cmd, err := h.CreateProcess(executable, execArgs...)
+	if err != nil {
+		return "", err
+	}
+
+	cmd.Dir = workingDirectory
+	bytes, err := cmd.CombinedOutput()
+	capturedOutput := strings.TrimSpace(string(bytes))
+
+	if err != nil {
+		return capturedOutput, fmt.Errorf("Error running process: %w", err)
+	}
+	return capturedOutput, nil
+}
+
 func (h Helm) InstallWithValues(chart string, valuesFile string, namespace string, release string) error {
 	var values []string
 	if valuesFile != "" {
 		values = []string{"--values", valuesFile}
 	}
 
-	if _, err := h.RunProcessAndCaptureOutput("helm", "install", release, chart, "--namespace", namespace,
-		"--wait", values, h.extraArgs); err != nil {
-		return err
+	helmArgs := []string{"install", release, chart, "--namespace", namespace, "--wait"}
+	helmArgs = append(helmArgs, values...)
+	helmArgs = append(helmArgs, h.extraArgs...)
+
+	if outputCapture, err := h.RunProcessAndCaptureOutput("helm", helmArgs); err != nil {
```

Comment:
This takes care of helm install command, what about helm test command? User would also want to see errors there right? See Test function below? No changes needed there?

Reply:
This has been fixed in d960d23, where both Helm and Kubectl types both use; Oc is covered in 
2c20705.

Reply:
The machinery responsible for enriching the error with the captured output content has been moved to another type and shared among all tools per d960d23 and bfb256e.